Parkduksung/ElevenStreetSearchExample/app/src/main/java/com/exam/elevenstreet/data/source/remote/ProductRemoteDataSource.kt

```diff
@@ -0,0 +1,74 @@
+package com.exam.elevenstreet.data.source.remote
+
+import com.exam.elevenstreet.network.RetrofitInstance
+import com.exam.elevenstreet.network.api.ElevenStreetApi
+import com.example.elevenstreet.ProductResponse
+import com.example.elevenstreet.ProductXmlPullParserHandler
+import java.net.URL
+
+class ProductRemoteDataSource {
+
+    private var elevenStreetApi: ElevenStreetApi? = null
+
+    fun getProductlist(Keyword: String, callback: (productList: List<ProductResponse>) -> Unit) {
```

Comment:
위와 동일합니다.
또한 변수명 규칙이 어긋납니다.
```suggestion
    fun getProductlist(keyword: String, callback: (productList: List<ProductResponse>) -> Unit) {
```

Reply:
`77b5fd8`  변경완료